apps/questions/templates/a4_candy_questions/question_module_detail.html

```diff
@@ -1,8 +1,132 @@
+                            data-toggle="tab"
+                            href="#tabpanel-module-{{ module.pk }}-information"
+                            role="tab"
+                            aria-controls="tabpanel-module-{{ module.pk }}-information"
+                            aria-expanded="false">
+                        {% trans 'Information' %}
+                    </a>
+                    <a
+                            id="tab-module-{{ module.pk }}-questions"
+                            class="tab active"
+                            data-toggle="tab"
+                            href="#tabpanel-module-{{ module.pk }}-questions"
+                            role="tab"
+                            aria-controls="tabpanel-module-{{ module.pk }}-questions"
+                            aria-expanded="true">
+                        {% trans 'Questions' %}
+                    </a>
+                    <a
+                            id="tab-module-{{ module.pk }}-statistics"
+                            class="tab"
+                            data-toggle="tab"
+                            href="#tabpanel-module-{{ module.pk }}-statistics"
+                            role="tab"
+                            aria-controls="tabpanel-module-{{ module.pk }}-module"
+                            aria-expanded="false">
+                        {% trans 'Statistics' %}
+                    </a>
+                </nav>
+            </div>
+        </div>
+
+        <div
+            class="tabpanel"
+            id="tabpanel-module-{{ module.pk }}-information"
+            role="tabpanel"
+            aria-labelledby="tab-module-{{ module.pk }}-information"
+            aria-expanded="false"
+        >
+            <div class="l-wrapper">
+                <div class="l-center-8">
+                    {{ module.description | transform_collapsibles | richtext }}
+                </div>
+            </div>
+        </div>
+
+        <div
+            class="tabpanel active"
+            id="tabpanel-module-{{ module.pk }}-questions"
+            role="tabpanel"
+            aria-labelledby="tab-module-{{ module.pk }}-questions"
+            aria-expanded="true"
+        >
+            <div class="container">
+                <div class="row mb-5">
+                    <div class="col-12">
+                        <div class="question-list">
+                            {% has_or_would_have_perm 'a4_candy_questions.change_question' request.user view.module as view_present_page %}
+                            {% if view_present_page %}
+                                <div class="d-flex justify-content-center justify-content-lg-end">
+                                    <a class="btn btn--secondary"
+                                       href="{% url 'question-present' module_slug=view.module.slug %}"
+                                       target="_blank"><span class="fa-stack fa-1x"><i class="fas fa-tv fa-stack-2x"
```

Comment:
I think indenting span and i on the < again would be more readable? But next time!